opsdroid/connector/matrix/connector.py

```diff
@@ -193,6 +193,8 @@ async def connect(self):
             encryption_enabled=self._allow_encryption,
             pickle_key="",
             store_name="opsdroid.db" if self._allow_encryption else "",
+            request_timeout=60,
+            max_timeouts=1,
```

Comment:
Neither of these new args are used anywhere?

Reply:
These arguments are given to the AsyncClientConfig from nio. Developers do not have to "use" it, as it is used directly by nio. See the [documentation](https://matrix-nio.readthedocs.io/en/latest/_modules/nio/client/async_client.html?highlight=AsyncClientConfig#), ctrl+F max_timeouts. 
request_timeout=60 is unnecessary as it is the default value, but max_timeouts prevents infinite retries.

Reply:
Oh I am sorry, I read this as the constructor to the class.

I think it would be good to add these to the connector config, so people can customise them?

Reply:
given the comments below about how `max_timeouts` work, perhaps just having `request_timeout` be configurable?

Reply:
I believe this kind of parameters should be shared between connectors, instead of a per-connector basis.
It is probably beyond the scope of this pull request.